### PR TITLE
If a location uses a booking location then populate details from that

### PR DESCRIPTION
**Pull request by stevenwilkin**
- opening hours
- phone number
- name of booking location

### CODE REVIEW

File: public/js/app.js [hunk 1]
```diff
@@ -133,6 +141,15 @@ function parseGeoJSON(geojson){
       location['lng'] = coordinates[0];
       location['lat'] = coordinates[1];
 
+      var bookingLocationId = location['booking_location_id'];
+
+      if(bookingLocationId && bookingLocationId != '') {
+        var bookingLocation = findLocationProperties(features, bookingLocationId);
```
Comment (gjvis) [inline public/js/app.js:147]: Perhaps a bit of defensiveness in case an invalid ID creeps in? (I know there are tests to prevent against it, but still...)

Comment (stevenwilkin) [inline public/js/app.js:147]: Wise idea!

**Comment by stevenwilkin** [comment]
Updated with some defensive programming

**Comment by gjvis** [comment]
it modifies state all over the shop :scream:, but it works, and its simple. :rocket: :shipit:

**Pull request merged**